streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java

```diff
@@ -51,21 +54,21 @@ public class GlobalExceptionHandler {
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     public RestResponse handleException(Exception e) {
         log.info("Internal server error：", e);
-        return RestResponse.fail("internal server error: " + e.getMessage(), ResponseCode.CODE_FAIL);
+        return RestResponse.fail(INTERNAL_SERVER_ERROR_ARGS, e.getMessage());
```

Comment:
`RestResponse.fail(...)` In many cases, a very specific business exception is returned, in short, the exception information of this error cannot be extracted uniformly and predetermined in advance.